src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/CertificateRevocationListBuilder.Load.cs

```diff
@@ -0,0 +1,286 @@
+                            AsnValueReader crlNumberReader = new AsnValueReader(
+                                extnValue,
+                                AsnEncodingRules.DER);
+
+                            crlNumber = crlNumberReader.ReadInteger();
+                            crlNumberReader.ThrowIfNotEmpty();
+                        }
+                    }
+                }
+
+                tbsCertList.ThrowIfNotEmpty();
+
+                while (revokedCertificates.HasData)
+                {
+                    RevokedCertificate revokedCertificate = new RevokedCertificate(ref revokedCertificates, version);
+                    list.Add(revokedCertificate);
+                }
+            }
+            catch (AsnContentException e)
+            {
+                throw new CryptographicException(SR.Cryptography_Der_Invalid_Encoding, e);
+            }
+
+            bytesConsumed = payloadLength;
+            currentCrlNumber = crlNumber;
+            return new CertificateRevocationListBuilder(list);
+        }
+
+        /// <summary>
+        ///   Decodes the specified Certificate Revocation List (CRL) and produces
+        ///   a <see cref="CertificateRevocationListBuilder" /> with all of the revocation
+        ///   entries from the decoded CRL.
+        /// </summary>
+        /// <param name="currentCrl">
+        ///   The PEM-encoded CRL to decode.
+        /// </param>
+        /// <param name="currentCrlNumber">
+        ///   When this method returns, contains the CRL sequence number from the decoded CRL.
+        ///   This parameter is treated as uninitialized.
+        /// </param>
+        /// <returns>
+        ///   A new builder that has the same revocation entries as the decoded CRL.
+        /// </returns>
+        /// <exception cref="ArgumentNullException">
+        ///   <paramref name="currentCrl" /> is <see langword="null" />.
+        /// </exception>
+        /// <exception cref="CryptographicException">
+        ///   <para>
+        ///     <paramref name="currentCrl" /> could not be decoded.
+        ///   </para>
+        ///   <para>- or -</para>
+        ///   <para>
+        ///     <paramref name="currentCrl" /> decoded successfully, but decoding did not
+        ///     need all of the bytes provided in the array.
+        ///   </para>
+        /// </exception>
+        public static CertificateRevocationListBuilder LoadPem(string currentCrl, out BigInteger currentCrlNumber)
+        {
+            ArgumentNullException.ThrowIfNull(currentCrl);
+
+            return LoadPem(currentCrl.AsSpan(), out currentCrlNumber);
+        }
+
+        /// <summary>
+        ///   Decodes the specified Certificate Revocation List (CRL) and produces
+        ///   a <see cref="CertificateRevocationListBuilder" /> with all of the revocation
+        ///   entries from the decoded CRL.
+        /// </summary>
+        /// <param name="currentCrl">
+        ///   The PEM-encoded CRL to decode.
```

Comment:
Would be nice if somewhere in the docs (and the string overload) we documented what PEM label we are looking for, as we have done with other PEM loading APIs.

_A PEM-encoded X.509 CRL EncryptedPrivateKeyInfo will begin with \<c>-----BEGIN X509 CRL-----\</c> and end with \<c>-----END X509 CRL-----\</c>, with the base-64 encoded DER contents of the key between the PEM boundaries._
        
Typically I stuck something like that in the `<remarks>`.